src/elements/content-sidebar/activity-feed/common/activity-message/CollapsableMessage.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import ToggleMoreLessButton from './ToggleMoreLessButton';
+import './ActivityMessage.scss';
+
+type Props = {
+    children: React.Node,
+};
+
+// this is for backward support for IE11
```

Comment:
We can remove this logic, if this is the case, since new features do not have to support IE11.